docusaurus/docs-internals/Icons/icons.md

```diff
@@ -1 +1,1579 @@
+---
+toc_max_heading_level: 4
+---
 # Icons
+
+import { SvgComponent } from '@site/src/components/SvgComponent';
+
+#### Usage
+To specify which `icon` you want to include in the control, pass the icon's name to the icon parameter in the control definition.
```

Comment:
add also that below is example:
`To specify which `icon` you want to include in the control, pass the icon's name to the icon parameter in the control definition. <br/>  Example: `